bin/opm

```diff
@@ -1345,7 +1345,7 @@ sub install_target ($$) {
         shell "rm -rf '$dist_dir'";
     }
 
-    shell "tar -xzf '$dist_file'";
+    shell "tar --no-same-owner -xzf '$dist_file'";
```

Comment:
Unfortunately, the `--no-same-owner` option is not portable at all. For example, it does not work on OS X:

```
$ tar -xvf --no-same-owner foo.tar.gz
tar: Error opening archive: Failed to open '--no-same-owner'
```

We need a portable way here.